miranda/convert/_rechunk.py

```diff
@@ -8,17 +8,35 @@
 import xarray as xr
 import zarr
 
-from miranda.ecmwf import ecmwf_variables
 from miranda.scripting import LOGGING_CONFIG
 
-from ._data import project_institutes
+from . import reanalysis_project_institutes
+from ._data_definitions import era5_variables
 
 logging.config.dictConfig(LOGGING_CONFIG)
 
 
 __all__ = ["rechunk_reanalysis"]
 
 
+def _rechunk_configurator(project, time_step):
```

Comment:
These are chunks saved to disk right?

Reply:
These chunks are saved to disk, currently as zarr files. We can adjust these chunk as soon as we have confirmation.

Reply:
I spoke with @aulemahal on this, and it seems as though chunk definition is something that needs to be determined depending on the use-case and the output format (netCDF or zarr). As it stands, these are the chunks that best describe data being downloaded at regular intervals for zarr, but it might be the case that we would want to eventually do something like creating netCDFs of chunks of four years to deal with the leap-day issue.

Ultimately, the choice of chunk will have everything to do with the data that we are comparing it to and/or the service that we are making it available through. For intermediary purposes, these chunks seem to be working for writing the output zarr. We can go from there.

@tlogan2000 If everything is good, I want to say this is resolved for the time being.

Reply:
ok for me